### ISSUE TITLE
Module installation

### ISSUE THREAD

**Issue by mbaudin47** [openturns/otmorris#43]
Installing the otmorris module is always a nightmare, because there is only a conda package. 

1) Installing it requires to install the openturns version from conda, which is very long. 

2) To make the plots work, another module is required, which is not automatically installed:

```
sudo apt-get install dvipng
```

3) Even with this setup, I have the following error that I do not understand:

```
$ conda install -c conda-forge otmorris
Collecting package metadata (current_repodata.json): done
Solving environment: done
==> WARNING: A newer version of conda exists. <==
  current version: 4.8.3
  latest version: 4.9.2
Please update conda by running
    $ conda update -n base -c defaults conda
# All requested packages already installed.
$ python
Python 3.8.3 (default, May 19 2020, 18:47:26) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import otmorris
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'otmorris'
```

My questions:
* What is the cause of that error? 
* Is there any way to create a pip package? It would be *much* easier and faster.

**Comment by mbaudin47** [comment]
I got the bug. There are several Python engines on my virtual environment. The one that contains otmorris is Python3.9:
```$ python3.9
Python 3.9.1 | packaged by conda-forge | (default, Jan 26 2021, 01:34:10) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import otmorris
```
But otmorris remains unknown to Jupyter:

![image](https://user-images.githubusercontent.com/31351465/107146927-71582d80-694b-11eb-8608-84898c27f9f9.png)

**Comment by mbaudin47** [comment]
I had it work with a completely fresh conda install:
```
wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O $HOME/miniconda.sh
bash $HOME/miniconda.sh -b -p $HOME/condaOTMorris
export PATH="$HOME/condaOTMorris/bin:$PATH"
source $HOME/miniconda/bin/activate
conda config --add channels conda-forge
conda config --set channel_priority strict
conda update -y conda
conda install -y openturns=1.16
conda install -y otmorris conda install
conda install -y scipy matplotlib jupyter
sudo apt-get install dvipng # For otmorris
```
I admit I am too stupid to manage my conda virtual environments correctly! 
However, a pip package would make it much simpler. 
Are you OK for me to create and maintain one?

**Comment by sofianehaddad** [comment]
ok for me. @jschueller ?

**Comment by jschueller** [comment]
I think no one is working on modules pip packages, so you can go ahead.
Are you sure you need dvipng there ?

**Comment by mbaudin47** [comment]
Some graphics error was produced without `dvipng` (but I did not, unfortunately, wrote the error message in my notes).

As for pip packages, here are the existing packages related to OpenTURNS:
* https://pypi.org/project/shapley-effects/
* https://pypi.org/project/otbenchmark/
* https://pypi.org/project/othdrplot/
* https://pypi.org/project/openmdao-extensions/
* https://pypi.org/project/impedancefitter/

I did not know of the last two packages. 

So I will try to create a otmorris package ASAP.

**Comment by jschueller** [comment]
I updated the conda install instructions in openturns.

We have pip packages but unfortunately for now they have to be downloaded manually from https://github.com/openturns/build-wheels/releases/tag/v1.17rc1, eg:
```
pip3 install https://github.com/openturns/build-wheels/releases/download/v1.17rc1/otmorris-0.10-cp38-cp38-manylinux1_x86_64.whl  https://github.com/openturns/build-wheels/releases/download/v1.17rc1/openturns-1.17rc1-cp38-cp38-manylinux1_x86_64.whl
```
It will be available on pypi for the 1.17

**Comment by jschueller** [comment]
they're available now (not osx though)
